lldb/source/API/SBDebugger.cpp

```diff
@@ -1667,6 +1667,12 @@ SBTypeSynthetic SBDebugger::GetSyntheticForType(SBTypeNameSpecifier type_name) {
       DataVisualization::GetSyntheticForType(type_name.GetSP()));
 }
 
+void SBDebugger::ResetStatistics() {
```

Comment:
Should we add a `void SBTarget::ResetStatistics()` too that calls with the right target?